FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,47 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 	
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
+		client.get(from: url, completion: { [ weak self ] result in
+			guard self != nil else { return }
+			switch result {
+			case .failure:
+				completion(.failure(Error.connectivity))
+			case .success((let data, let response)):
+				completion(FeedImageMapper.map(from: data, response))
+			}
+		})
+	}
+	
+	private class FeedImageMapper {
+		static let OK_200: Int = 200
```

Comment:
The `OK_200` property should have a private access level since it is used only in the `FeedImageMappper`.

```suggestion
		private static let OK_200: Int = 200
```

Reply:
Updated